swaglyrics/__main__.py

```diff
@@ -28,12 +28,19 @@ def main():
 	# 	""")
 	# print('\n')
 
+	program = "Swaglyrics"
 	parser = argparse.ArgumentParser(
+		prog=program,
```

Comment:
So I checked and since the default value of `prog` is `sys.argv[0]` this is unneeded

Reply:
What's the value of `sys.argv[0]`?

Reply:
https://docs.python.org/3/library/sys.html